app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -55,14 +49,10 @@ def update
         sca_param = ::Foreman::Cast.to_bool(params[:simple_content_access])
         if sca_param && !@taxonomy.simple_content_access?
           # user has requested SCA enable
-          task = async_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:id])
-          info "Enabling Simple Content Access for organization #{@taxonomy.name}.",
-            link: { text: "View progress on the Tasks page", href: "/foreman_tasks/tasks/#{task&.id}" }
+          async_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:id])
```

Comment:
```suggestion
          sync_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:id])
```
I think this is quick enough that these can be sync tasks anyway